repairnator/kubernetes-support/README.md

```diff
@@ -0,0 +1,174 @@
+# Deploy Repairnator in Kubernetes
+Note: to store data you can either deploy activeMQ on cloud described at the bottom of this README or if you have a mongodb outside the cluster you can map it as an external service named as mongo (since all of the default mongo k8s service name is named "mongo" if you don't like a lot changes). More details for how this is done is described in [this link](ttps://cloud.google.com/blog/products/gcp/kubernetes-best-practices-mapping-external-services)
+
+![Design](K8sRepairnatorDesign.jpg)
+
+## Prerequisites
+
+* kubectl
+* a working k8s cluster (preferable high cpu capacity)
+* gcloud (for creating the cluster mentoned above and creating mongodb if you don't have them already)
+
+## Setup ActiveMQ
+
+To deploy ActiveMQ for effective queue managements of jobs to the pipeline and scanner. First apply the yaml file inside the queue-for-buildids folder
+```
+	kubectl create -f /queue-for-buildids/activemq.yaml
```

Comment:
where is this file?

Reply:
I have included the file in the latest commit. :)

Reply:
```suggestion
    cd  repairnator/kubernetes-support/
	kubectl create -f ./queue-for-buildids/activemq.yaml
```

Reply:
fixed :) .